spec.bs

```diff
@@ -1658,24 +1660,32 @@ An interest group is a [=struct=] with the following items:
   "<code>compatibility</code>" and "<code>group-by-origin</code>".
 : <dfn>bidding url</dfn>
 :: Null or a [=URL=]. The URL to fetch the buyer's JavaScript from.
-  <p class="note">The [=interest group/bidding url=] [=origin=] will always be [=same origin=] with
-  [=interest group/owner=].</p>
+  <p class="note">
+    The [=interest group/bidding url=] [=origin=] will always be [=same origin=] with
+    [=interest group/owner=].
+  </p>
 : <dfn>bidding wasm helper url</dfn>
 :: Null or a [=URL=]. Lets the bidder provide computationally-expensive subroutines in WebAssembly,
   in addition to JavaScript, to be driven from the JavaScript function provided by
   [=interest group/bidding url=].
-  <p class="note">The [=interest group/bidding wasm helper url=] [=origin=] will always be [=same origin=] with
-  [=interest group/owner=].</p>
+  <p class="note">
+    The [=interest group/bidding wasm helper url=] [=origin=] will always be [=same origin=] with
+    [=interest group/owner=].
+  </p>
 : <dfn>update url</dfn>
 :: Null or a [=URL=]. Provides a mechanism for the group's owner to periodically update the
   attributes of the interest group. See [interest group updates](#interest-group-updates).
-  <p class="note">The [=interest group/update url=] [=origin=] will always be [=same origin=] with
-  [=interest group/owner=].</p>
+  <p class="note">
+    The [=interest group/update url=] [=origin=] will always be [=same origin=] with
+    [=interest group/owner=].
+  </p>
 : <dfn>trusted bidding signals url</dfn>
 :: Null or a [=URL=]. Provide a mechanism for making real-time data available for use at bidding
   time. See [=building trusted bidding signals url=].
-  <p class="note">Note: The [=interest group/trusted bidding signals url=] [=origin=] will always be [=same origin=] with
-  [=interest group/owner=].</p>
+  <p class="note">
+    The [=interest group/trusted bidding signals url=] [=origin=] will always be [=same origin=]
+    with [=interest group/owner=].
+  </p>
 : <dfn>trusted bidding signals keys</dfn>
 :: Null or a [=list=] of [=string=]. See [=building trusted bidding signals url=].
 : <dfn>user bidding signals</dfn>

```

Comment:
(several lines down -- Github doesn't let me put a comment there):

<dfn>next update after</dfn> should probably link to the update section?